Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "AxisArrays"
 uuid = "39de3d68-74b9-583c-8d2d-e117c070f3a9"
-version = "0.3.3"
+version = "0.3.4"
```

Comment:
I was about to hit merge and then remembered, are we allowed to narrow the range of supported Julia versions on a patch release?

Reply:
You are correct :( we are not. I'll bump to 0.4.0.

Reply:
Done.